cla_backend/apps/reports/tests/test_reports.py

```diff
@@ -261,8 +261,7 @@ def tearDown(self):
 
 
 class TestKnowledgeBaseArticlesExport(TestCase):
-    @classmethod
-    def setUpClass(cls):
+    def setUp(self):
```

Comment:
"setUp runs before each test in a test case at the instance level. setUpTestData runs once before each test case at the class level. So if you have a TestCase with N tests, setUpTestData will be called only once (before *all* tests) and setUp will be called N times (before *each* test)."

So I suppose it is down to how we want this test to run. 


https://docs.djangoproject.com/en/4.1/topics/testing/tools/#django.test.TestCase.setUpTestData
https://docs.python.org/3/library/unittest.html#unittest.TestCase.setUp

Reply:
I've changed it to setUpTestData() and kept the @classmethod decorator in there. This is to mimic the prior setUpClass() functionality. This is also in line with the change we made in cla_backend/apps/call_centre/tests/api/test_case_api.py